cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AddressMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import uk.gov.hmcts.cmc.domain.models.Address;
+
+public class AddressMapper {
+
+    uk.gov.hmcts.cmc.ccd.domain.Address toCCD(Address address) {
+
+        uk.gov.hmcts.cmc.ccd.domain.Address ccdAddress = uk.gov.hmcts.cmc.ccd.domain.Address
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "ccdAddress". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1488)